src/components/Molecules/Promo/Promo.md

```diff
@@ -173,12 +169,12 @@ let mobileVideoSrc = 'https://samplelib.com/lib/preview/mp4/sample-10s.mp4';
   imageLow={defaultData.promoImage}
   imageAltText=""
   copyLeft={true}
-  hasOverlay={true}
   position="upper"
   videoSrc={videoSrc}
   poster={poster}
   loop={true}
   autoplay={true}
+  behindTextGradient='black'
```

Comment:
While these examples are set up to show loads of different combinations of the various settings, within CRcom, there'll be some logic to handle these; 

i.e, when copy is set to 'white', it'll programmatically assign a black gradient overlay and a black play button.